### ISSUE TITLE
Can you share the training parameters? I am using a dataset and cannot obtain good results

### ISSUE THREAD

**Issue by yangh-zzf-itcast** [ridvansalihkuzu/vein-biometrics#12]
this is my parameters 
Namespace(batch_size=32, database_dir='/ssd_data/yanghang/palm_vein/POLYU/NIR_polyu_crop/', embedding_size=512, evaluate=False, learning_rate=0.01, logdir='./data_polyup/CSVFiles/cnn_logs.csv', margin=0.5, num_classes=450, num_epochs=300, num_workers=8, outdir='modeldir/00/05/', scale_rate=32.0, start_epoch=0, test_dir='/ssd_data/yanghang/palm_vein/POLYU/data_polyup_CSVFiles/output_list_test_pair.csv', train_dir='/ssd_data/yanghang/palm_vein/POLYU/data_polyup_CSVFiles/output_list_train.csv', type='aamp', valid_dir='/ssd_data/yanghang/palm_vein/POLYU/data_polyup_CSVFiles/output_list_val.csv', weight_decay=0.0001)
I use polyu palm vein dataset，and use MNASNet_Modified model，The validation set loss keeps rising, obviously overfitting
![image](https://user-images.githubusercontent.com/45489193/233940249-7633b11e-dbec-44c0-85fd-dd7dbbaaf86f.png)

**Comment by 151210148** [comment]
hi , POLYU data can share? thanks

**Comment by jieblue** [comment]
Can u share POLYU palm vein datasets？ Thanks.

**Comment by deeperlearner** [comment]
@yangh-zzf-itcast 
I have the same problem. Validation prec stays at zero, but the test pair results seem good.
I don't think the training process is responsible for the good test results. This is due to the pre-trained weights of ImageNet's MNASNet_Modified model.
Do you have any findings? Thanks.
![image](https://github.com/user-attachments/assets/11fdba71-b2f7-4c0c-9765-e9f9f32df83b)

**Comment by deeperlearner** [comment]
Reduce the batch size can improve the validation performance.
`python3 -m benchmark_verification.main_train_CNN --num_epochs 100 --batch-size 8 --type none`
![image](https://github.com/user-attachments/assets/1883321c-0895-4888-8922-6ec7ae92e948)
